openbb_terminal/stocks/fundamental_analysis/polygon_view.py

```diff
@@ -0,0 +1,66 @@
+"""Polygon view"""
+__docformat__ = "numpy"
+import logging
+import os
+
+from openbb_terminal.decorators import check_api_key, log_start_end
+from openbb_terminal.helper_funcs import (
+    export_data,
+    lambda_long_number_format,
+    print_rich_table,
+)
+from openbb_terminal.rich_config import console
+from openbb_terminal.stocks.fundamental_analysis import polygon_model
+
+logger = logging.getLogger(__name__)
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_POLYGON_KEY"])
+def display_fundamentals(
+    ticker: str,
+    financial: str,
+    limit: int = 10,
+    quarterly: bool = False,
+    export: str = "",
+):
+    """Display tickers balance sheet
+
+    Parameters
+    ----------
+    ticker: str
+        Stock ticker
+    financial:str
+        Either balance or income
+    limit: int
+        Number of results to show
+    quarterly:bool
+        Flag to get quarterly reports
+    export: str
+        Format to export data\
+    """
+    if financial == "balance":
+        fundamentals = polygon_model.get_financials(ticker, quarterly)[0]
+        title_str = "Balance Sheet"
+    elif financial == "income":
+        fundamentals = polygon_model.get_financials(ticker, quarterly)[1]
+        title_str = "Income Statement"
+
+    if fundamentals.empty:
```

Comment:
Can we have an error message here?

Reply:
I do the errors in the model this way we know what went wrong instead of a general "something went wrong"

Reply:
The error message I mean "Fundamentals is empty" or so

Reply:
![image](https://user-images.githubusercontent.com/18151143/167263821-af07cbd9-4116-465e-93b8-c36bf4718295.jpeg)

Reply:
Gotcha gotcha. I'm slow today 😰